ReportingServicesTools/Functions/CatalogItems/Rest/New-RsCacheRefreshPlan.ps1

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT License (MIT)
+
+function New-RsRestCacheRefreshPlan
+{
+    <#
+        .SYNOPSIS
+            This function creates a new CacheRefreshPlan for the specified Power BI Report
```

Comment:
is this generic for any report or just for PBI reports? If is the former I would update the documentation

Reply:
We are using a URL which specifically includes "PowerBIReports" in the name.
> `"api/$RestApiVersion/PowerBIReports({0})/CacheRefreshPlans"`

Given the URL in use, I thought it would only work for a Power BI report?

Reply:
My testing has shown that the `New-RsRestCacheRefreshPlan` function is unable to create a CacheRefreshPlan for a paginated report; so the documentation is accurate as far as I can tell.